ccm_web/client/src/pages/Home.tsx

```diff
@@ -61,18 +95,14 @@ function Home (props: HomeProps): JSX.Element {
   }
 
   const renderCourseRename = (): JSX.Element => {
-    if (isAuthorizedForRoles(props.globals.course.roles, courseRenameRoles, 'Course Rename')) {
-      const saveCourseName = async (courseName: string): Promise<void> => await new Promise<void>((resolve, reject) => {
-        console.log('saveCourseName ' + courseName)
-        if (courseName === 'reject') {
-          reject(new Error('Rejected as requested'))
-        } else {
-          return resolve()
-        }
-      })
-      return (<InlineTextEdit {...{ text: 'Course 123ABC', save: saveCourseName, placeholderText: 'Course name', successMessage: 'Saved', failureMessage: 'Error saving course name' }}/>)
+    if (isLoadingCourse) {
+      return (<div className={classes.courseName}>Loading...</div>)
+    } else if (getCourseError !== undefined || course === undefined) {
+      return (<div className={classes.courseName}>Error getting course name</div>)
```

Comment:
I think this is okay for now, but we probably want to look at more robust and re-usable loading (spinner?) and error displays. Seems like a separate issue.

Reply:
Agreed, the loading text is gone before I can even see it, so it might be better off just not showing anything rather than worrying about a spinner.

Reply:
I feel @ssciolla suggested for a spinner and @chrisrrowland don't want that but leave it blank. I think there is some disagreement, unless I did not understood the comments.

I feel the spinner is there currently until get the course name

Reply:
I think the spinner question can be discussed/handled in a separate issue. I'm just saying, instead of having separate (or separately styled) loading or error displays, we probably want to adopt a generic approach for that. It'll become more relevant once we're making other API calls. I'm fine with what @chrisrrowland has here for the time being.